### PR TITLE
[Webview] Fix load HTML from string for Android 10

### PR DESCRIPTION
**Pull request by GuillaumeSE**
GitHub Issue (If applicable): #
https://github.com/unoplatform/uno/issues/4446 

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?

Android 10's Webview does not display html content from string properly. But work fine on Android 9
Screenshot_20201112-123711_iA_Mobile 1

## What is the new behavior?

The content is properly displayed from html string for Android 9 and upper
image

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Ref: https://developer.android.com/reference/android/webkit/WebView#loadData(java.lang.String,%20java.lang.String,%20java.lang.String)
Internal Issue (If applicable):

### CODE REVIEW

**Comment by jeromelaban** [comment]
Thanks ! Would you be able to create a UI or Runtime Test for that ? You can set a body with a specific color and assert that the color inside the webview is the right one.

**Comment by davidjohnoliver** [comment]
> Thanks ! Would you be able to create a UI or Runtime Test for that ? You can set an HTML body with a specific color and assert that the color inside the webview is the right one.

Docs on creating runtime tests: https://platform.uno/docs/articles/uno-development/creating-runtime-tests.html

Or ping us if you have any questions :)

File: src/Uno.UI/UI/Xaml/Controls/WebView/WebView.Android.cs [hunk 1]
```diff
@@ -134,7 +134,8 @@ private void CreateAndLaunchMailtoIntent(Android.Content.Context context, string
 			}
 
 			_wasLoadedFromString = true;
-			_webView.LoadData(text, "text/html; charset=utf-8", "utf-8");
+			//Note : _webView.LoadData does not work properly on Android 10 even when we encode to base64.
+			_webView.LoadDataWithBaseURL(null, text, "text/html; charset=utf-8", "utf-8", null);
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/WebView/WebView.Android.cs:138]: does this work correctly on older versions of Android?

Comment (GuillaumeSE) [reply src/Uno.UI/UI/Xaml/Controls/WebView/WebView.Android.cs:138]: Tested on simulators from Android 8 to 11 and it's working well.
I will try on more older to be sure

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/WebView/WebView.Android.cs:138]: With a UI Test, we'll validate on 5 and 11

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/WebView/WebView.Android.cs:138]: It's already tested indeed, once we add testing for Android11, it'll be validated as well.

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/3.1

**Pull request merged**